terraform/modules/g-drive-to-s3/10-lambda.tf

```diff
@@ -138,6 +138,7 @@ resource "aws_lambda_function" "g_drive_to_s3_copier_lambda" {
   s3_bucket        = var.lambda_artefact_storage_bucket
   s3_key           = aws_s3_object.g_drive_to_s3_copier_lambda.key
   source_code_hash = data.archive_file.lambda.output_base64sha256
+  layers           = ["arn:aws:lambda:eu-west-2:${data.aws_caller_identity.current.account_id}:layer:google-apis-layer:1"]
```

Comment:
I was going to recommend referencing the resource from the module output, but it doesn't output the arns - I'll make a note to go back and do that at some point.

Reply:
Yeah, I've noticed that too. When I am creating a pull request for Marta's lambda, I will output the ARNs of all the layers